src/test/java/stepdefs/FiltersPageStepDefs.java

```diff
@@ -0,0 +1,51 @@
+package stepdefs;
+
+import buisness.components.AddFilterComponent;
+import buisness.core.FiltersPage;
+import buisness.core.LauncherPage;
+import io.cucumber.datatable.DataTable;
+import io.cucumber.java.en.Then;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.RandomStringUtils;
+
+import java.util.List;
+
+import static org.junit.Assert.assertTrue;
+
+@Slf4j
+public class FiltersPageStepDefs {
+  private final FiltersPage filtersPage = new FiltersPage();
+  private final LauncherPage launcherPage = new LauncherPage();
+  private final AddFilterComponent addFilterComponent = new AddFilterComponent();
+
+  @Then("Filters grid contains such filters")
+  public void filterGridContainsSuchFilters(final DataTable dataTable) {
+    filtersPage.filtersPageShouldBeOpened();
+    filtersPage.gridShouldContainsFilters();
+    List<String> filterNames = dataTable.asList();
+    filterNames.forEach(e ->
+      assertTrue(filtersPage.filterByNameShouldBeVisible(e))
```

Comment:
it would be nice to add error message to simplify debug in future

Reply:
thanks, added assert with a message